src/test/java/org/opentripplanner/model/plan/legreference/LegReferenceSerializerTest.java

```diff
@@ -1,25 +1,38 @@
 package org.opentripplanner.model.plan.legreference;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 import java.time.LocalDate;
 import org.junit.jupiter.api.Test;
+import org.opentripplanner.transit.model._data.TransitModelForTest;
 import org.opentripplanner.transit.model.framework.FeedScopedId;
 
 class LegReferenceSerializerTest {
 
-  private static final FeedScopedId TRIP_ID = new FeedScopedId("F", "Trip");
+  private static final FeedScopedId TRIP_ID = TransitModelForTest.id("Trip");
   private static final LocalDate SERVICE_DATE = LocalDate.of(2022, 1, 31);
   private static final int FROM_STOP_POS = 1;
+
+  private static final FeedScopedId FROM_STOP_ID = TransitModelForTest.id("Boarding Stop");
   private static final int TO_STOP_POS = 3;
+  private static final FeedScopedId TO_STOP_ID = TransitModelForTest.id("Alighting Stop");
   private static final String ENCODED_TOKEN =
-    "rO0ABXc2ABhTQ0hFRFVMRURfVFJBTlNJVF9MRUdfVjEABkY6VHJpcAAKMjAyMi0wMS0zMQAAAAEAAAAD";
+    "rO0ABXdZABhTQ0hFRFVMRURfVFJBTlNJVF9MRUdfVjEABkY6VHJpcAAKMjAyMi0wMS0zMQAAAAEAAAADAA9GOkJvYXJkaW5nIFN0b3AAEEY6QWxpZ2h0aW5nIFN0b3A=";
```

Comment:
Is this the token without the stop id? If so, I'd appreciate a comment indicating this.

Reply:
This is a token with the new encoding format (including stop ids)
Commented.